README.md

```diff
@@ -49,9 +49,9 @@ or
 | provisionvenom_policy_description | The description to associate with the IAM policy that allows provisioning of the VENOM layer in the Shared Services account. | `string` | `Allows provisioning of the VENOM layer in the Shared Services account.` | no |
 | provisionvenom_policy_name | The name to assign the IAM policy that allows provisioning of the VENOM layer in the Shared Services account. | `string` | `ProvisionVenom` | no |
 | tags | Tags to apply to all AWS resources created. | `map(string)` | `{}` | no |
-| venom_cidrs | A map with keys equal to the VENOM CIDR blocks and values equal to a brief description (e.g. {"10.200.0.0/16": "Primary", "10.201.0.0/16": "Secondary"}). | `map(string)` | n/a | yes |
+| venom_cidrs | A map with values equal to the VENOM CIDR blocks and keys equal to a brief description (e.g. {"Primary": "10.200.0.0/16", "Secondary": "10.201.0.0/16"}). | `map(string)` | n/a | yes |
```

Comment:
if we're going to use a hard-coded key per
https://github.com/cisagov/cool-sharedservices-venom/blob/f8591be1f40eb2457a9402859027c5fde58367ea/transitgateway.tf#L33
and
https://github.com/cisagov/cool-sharedservices-venom/blob/f8591be1f40eb2457a9402859027c5fde58367ea/transitgateway.tf#L43
, then I think we should use that in the description (or a call out in the README somewhere). There's no indication here that a specific key is required.

Reply:
That prefix list was a failed attempt to pre-emptively support VENOM-East and VENOM-West.  I can't really do that properly right now since VENOM-West doesn't exist yet, so I removed the prefix list in commit a0c4708 and am using a simple CIDR block instead.

Reply:
Ahh, thanks for the background information.